### PR TITLE
Show clear and actionable query timeout error message

### PR DESCRIPTION
**Pull request by graceguo-supercat**
1. Instead of waiting for a long time or server-side response 504 Gateway timeout, explore view now add a query timeout threshold. After timeout it will show specific query timeout warning.
2. fix alert box close button position.

https://github.com/airbnb/superset/issues/2734

before
before

after
after

### CODE REVIEW

**Comment by ascott** [comment]
@graceguo-supercat could you plz add before/after screenshots?

File: superset/assets/javascripts/constants.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const QUERY_TIMEOUT_THRESHOLD = 60000;
```
Comment (ascott) [inline superset/assets/javascripts/constants.js]: seems like this should live in the sql lab constants file https://github.com/airbnb/superset/blob/dd4b2a5e2603e553c28c2967fb2d37b7d031510a/superset/assets/javascripts/SqlLab/constants.js

File: superset/assets/javascripts/modules/utils.js [hunk 2]
```diff
@@ -189,7 +189,7 @@ export function customizeToolTip(chart, xAxisFormatter, yAxisFormatters) {
   });
 }
 
-export function initJQueryAjaxCSRF() {
+export function initJQueryAjax() {
```
Comment (ascott) [inline superset/assets/javascripts/modules/utils.js]: curious why the function name is changed in this PR?

**Comment by ascott** [comment]
LGTM, thanks @graceguo-supercat!

File: superset/assets/javascripts/modules/utils.js (hunk 2 above)
Comment (graceguo-supercat) [inline superset/assets/javascripts/modules/utils.js]: there are actually multiple places need to handle 504 Gateway timeout problem for long query (but this ticket only fix for explore view). 

At the beginning I want to set a default jQuery ajax timeout limit from ajaxSetup, for all dashboard and explore view queries. It seems currently we only used this setup for CSRF, so it named as initJQueryAjaxCSRF. I am just thinking we might need a function to setup global, default jQuery ajax related settings, and this function is already hooked-up by many components.

Comment (ascott) [inline superset/assets/javascripts/modules/utils.js]: makes sense!

**Pull request merged**